examples/CSharp/Tutorial 1.4 - Textures/Shader.cs

```diff
@@ -53,7 +52,6 @@ public void SetUniform(string name, float value)
             {
                 throw new Exception($"{name} uniform not found on shader.");
             }
-            Use();
```

Comment:
Have you made sure we've definitely called UseProgram before we use any of the SetUniform functions? While we don't need to do it every call, we need to at least make sure it's called once prior to the uniform calls.

Reply:
Yes, in the OnRender method for each tutorial (or RenderLitCube/RenderLamp for a couple of the later ones) we have a Shader.Use() call immediately before SetUniform

Reply:
Awesome, thanks.